docs/reference/search/field-caps.asciidoc

```diff
@@ -78,13 +78,28 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailab
   Defaults to `false`.
 
 `filters`::
-(Optional, string) Comma-separated list of filters to apply to the response.  The
-following filters are supported: +metadata,-metadata,-parent,-nested,-multifield
+(Optional, string) Comma-separated list of filters to apply to the response.
++
+.Valid values for `filters`
+[%collapsible%open]
+====
+`+metadata`::
+Only include metadata fields
+`-metadata`::
+Exclude metadata fields
+`-parent`::
+Exclude parent fields
+`-nested`::
+Exclude nested fields
+`-multifield`::
+Exclude multifields
+====
```

Comment:
is parent without operator accepted after all? Sorry I keep on getting confused about this.

Reply:
It's accepted for the moment but not documented, I can remove it entirely if you prefer?

Reply:
would be nice to remove unless anybody relies on it? Would it make sense to expand the docs on the fact that we only ever support exclusions (besides one case)? That looks weird at first glance.

Reply:
I am not sure yet, I am open to ideas. I find it peculiar that you have a filters parameter and you can only ever use it to exclude things. That should be ok in case we will need to add inclusion in the future, otherwise we could rename the parameter to something more inline with exclusions.

Reply:
I forgot that we also have +metadata, and not only - , ok let's leave things as-is for now. We can always add new filters later and these are the bare minimum that consumers we'll need.